components/organisms/CriteriaGrid.js

```diff
@@ -0,0 +1,120 @@
+import PropTypes from "prop-types";
+import { CriteriaBox } from "../atoms/CriteriaBox";
+import { SelectPicker } from "../atoms/SelectPicker";
+import { NumInput } from "../atoms/NumInput";
+
+import { useTranslation } from "next-i18next";
+
+export function CriteriaGrid(props) {
```

Comment:
This grid looks great! My only concern is that it's essentially hardcoded with the three CriteriaBoxes and none of the props being passed in are being used.